crates/re_query_cache/tests/range.rs

```diff
@@ -0,0 +1,388 @@
+//! Contains:
+//! - A 1:1 port of the tests in `crates/re_query/tests/archetype_range_tests.rs`, with caching enabled.
```

Comment:
it would be nice to share some of store setup code from there

Reply:
Eh, I do expect them to grow apart as time goes on; they don't worry about the same things exactly.